flocker/node/agents/functional/test_gce.py

```diff
@@ -24,23 +24,21 @@
 """
 
 from uuid import uuid4
+from bitmath import GiB
+
 from fixtures import Fixture
 from characteristic import attributes
 
-from ..blockdevice import AlreadyAttachedVolume
-
-from ..gce import get_machine_zone, get_machine_project
+from ..blockdevice import AlreadyAttachedVolume, MandatoryProfiles
+from ..gce import get_machine_zone, get_machine_project, GCEDiskTypes
 from ....provision._gce import GCEInstanceBuilder
-
 from ..test.test_blockdevice import (
-    make_iblockdeviceapi_tests
+    make_iblockdeviceapi_tests, make_iprofiledblockdeviceapi_tests,
```

Comment:
Maybe remove this import and add an explanatory note if those tests can't be used for some reason.